legacy/jacorb/src/main/java/org/jboss/as/jacorb/JacORBSubsystemParser.java

```diff
@@ -680,7 +680,7 @@ private void parseGenericProperty_1_0(XMLExtendedStreamReader reader, ModelNode
         if (!required.isEmpty()) {
             throw missingRequired(reader, required);
         }
-        node.add(name, val);
+        node.get(name).set(val);
```

Comment:
Because these are "properties" attributes, so the code that reads them is going to do node.asPropertyList() and that works just fine with the LIST of PROPERTY produced by the existing code.

But WFCORE-1581 means the AD is going to try and store them in their "spec" form as ModelType.OBJECT. That will work fine even without this change. But the transformer tests fail because they compare the param to the model and ModelType.LIST != ModelType.OBJECT.